docs/docs/guides/how-to-add-digest-to-email-notifications.md

```diff
@@ -0,0 +1,259 @@
+- **5-Custom Code** - In custom code, you can write custom digest templates to show some/all parts of a message. Read [more about it here](https://docs.novu.co/platform/digest#writing-digest-templates).
+- **6-Test** - This allows you to send a test email to test if everything is working as per your wish.
+
+You can write your own digest template in the 'custom code' section or just follow this guide as shown in the picture above.
+Once you’re done configuring this to your liking, click on the ‘update’ button on the top right.
+
+It’ll automatically create a trigger code that you can use in your app. To get it, click on the ‘get snippet’ button on the top right and copy it:
+
+![code snippet to trigger ](https://res.cloudinary.com/dxc6bnman/image/upload/v1689096592/guides/Screenshot_2023-07-11_at_10.59.42_PM_pmd1as.png)
+
+Now, let’s see how to add Novu to our app!
+
+## Add Novu to the Backend and Configure it
+
+In your backend, install the novu package using the following code:
+
+```bash
+npm install @novu/node
+```
+
+Now, create a route that you want to hit when called from the front end. In our demo app, this is the route:
+
+```jsx
+import express from "express";
+
+import { getEmailDigest } from "../controller/emaildigest.js";
+
+const router = express.Router();
+
+router.post('/sending-email-digest', getEmailDigest);
+
+export default router;
+```
+
+Now, we need to write a controller function that will handle the logic for what is to be sent in the trigger function’s payload. In our case, this is the controller function:
+
+```jsx
+import { sendEmailDigest } from "../novu/novu.js";
+
+export const getEmailDigest = async (req, res) => {
+    const { notif, email } = req.body
+    try {
+        await sendEmailDigest(notif, email);
+        res.status(201).json({ message: 'success', notif: notif });
+    } catch (error) {
+        res.status(409).json({ message: error.message });
+    }
+}
+```
+
+To make it modular, we’ll keep the trigger code in a separate function in a separate file (’novu.js’, in our case) and the trigger function is getting called in the controller function above by the name ‘getEmailDigest’.
+
+If you’re following the guide, you’ve already copied the trigger function. But before we can add it to our app, we need one key thing - Subscribers.
+
+Subscribers are entities to which the notifications are sent. You can see a list of subscribers in the [Novu dashboard](https://web.novu.co/subscribers) as well.
+
+In our app, we’ll create a subscriber in Node.js as we’re writing our backend in Node.js, but we also have backend SDKs (Node.js, PHP, .NET, Go, Ruby, Python, and Kotlin) to choose from. The recommended way to create a subscriber in NodeJS is as follows:
```

Comment:
Can we link the SDKs here to all the individual Sdk repos here?

Reply:
added all ✅